src/app/component/user/user.module.ts

```diff
@@ -164,7 +165,7 @@ import { RequestItemComponent } from './components/profile/users-friends/friend-
     DragDropModule,
     HttpClientModule,
     AgmCoreModule.forRoot({
-      apiKey: 'AIzaSyB3xs7Kczo46LFcQRFKPMdrE0lU4qsR_S4',
+      apiKey: environment.agmCoreModuleApiKey,
```

Comment:
will it work for all languages?

Reply:
![Screenshot (545)](https://user-images.githubusercontent.com/50026335/117121709-a7b41600-ad9d-11eb-9451-ac2b2e0f4603.png)